### DISCUSSION TITLE
FR: I read a blog post but could not get my head around why I would choose this over Git

### DISCUSSION THREAD

**Discussion by alexec** [jj-vcs/jj#4027] [General]
I, like you, think the idea of as easy-to-use VCS would be wonderful. Yet, I cannot get my head around why I would use this instead of Git, I don't know Mercurial/Darcs/Spaling or any other VC system, so terms like "revset" are confusing for me. 

More docs sometimes is not helpful. Why about "Jujitsu for Git" article? These are things I do a lot:

- How to commit a change.
- How to make a branch.
- How to create a pull request.
- How to merge a pull request.
- How to fix merge conflicts.

**Comment by ilyagr** [comment]
The following articles have been helpful to various people:

https://steveklabnik.github.io/jujutsu-tutorial/ (the newest)

https://martinvonz.github.io/jj/latest/github/ (especially for Pull Requests)

An I'm guessing you've seen https://martinvonz.github.io/jj/latest/tutorial/ (the oldest).

I think they cover all of the topics you mention, but experience reports about what was confusing or helpful, as well as suggestions as to how to improve things are welcome.

Finally, to state the obvious, there is no obligation for you to wrap your head around `jj` or to use it over Git.  If something piques your curiosity, great! If not, perhaps something will interest you later, or perhaps `jj` just doesn't help with *your* workflow or the problems *you* have.
